stor/tests/test_posix_path_compat.py

```diff
@@ -179,7 +179,7 @@ def test_listing(self):
                 for f in files:
                     try:
                         f.remove()
-                    except:
+                    except Exception:
```

Comment:
This will no longer catch special Python exceptions https://stackoverflow.com/a/7161517. Is this expected?

Reply:
Yes in this case I don’t think we meant to catch KeyboardInterrupt or SystemExit :)